### PR TITLE
[privacy] fix: Restore mixed wallet with SPV on

### PR DESCRIPTION
**Pull request by amassarwi**
While working on #2907 found that restoring mixed wallet doesn't work with SPV on, this diff fixes that.

### CODE REVIEW

File: app/components/views/GetStartedPage/hooks.js [hunk 1]
```diff
@@ -208,9 +208,18 @@ export const useGetStarted = () => {
         if (synced === true) {
           send({ type: "GO_TO_HOME_VIEW" });
         }
-        if (context.isSPV) {
+        if (isSPV) {
           return startSPVSync(passPhrase)
-            .then(() => send({ type: "GO_TO_HOME_VIEW" }))
+            .then(() => {
+              if (isPrivacy) {
+                // if recoverying a privacy wallet, we go to settingMixedAccount
```
Comment (jholdstock) [inline app/components/views/GetStartedPage/hooks.js:215]:
```suggestion
                // if recovering a privacy wallet, we go to settingMixedAccount
```

File: app/components/views/GetStartedPage/hooks.js [hunk 2]
```diff
@@ -208,9 +208,18 @@ export const useGetStarted = () => {
...
+                // state, so the user can set a mixed account based on their
+                // coinjoin outputs.
+                // This state should only be achievable if recoverying wallet.
```
Comment (jholdstock) [inline app/components/views/GetStartedPage/hooks.js:218]:
```suggestion
                // This state should only be achievable if recovering wallet.
```

**Comment by jholdstock** [comment]
Looks good other than comment typos

**Comment by amassarwi** [comment]
@jholdstock This comment was deleted in https://github.com/decred/decrediton/pull/3033/files#diff-7703b3a83a5aa0b1e709012b1b720d618adcb22031c163096ac2d99e1af80925L236-L244 along with the `isPrivacy` check, so we can keep it as is in this PR (also to avoid a rebase conflict 🙏)

**Review by jholdstock — APPROVED** [review]

File: app/components/views/GetStartedPage/hooks.js (hunk 2 above)
Comment (vctt94) [inline app/components/views/GetStartedPage/hooks.js:218]: This state shouldn't be only achievable when recoverying a wallet, as the user may pass it and may want to do it later, but I am approving this PR, as this is true for this release, but we should allow it if one passes it, by mistake or whatever other reason.

**Review by vctt94 — APPROVED** [review]
Left a comment, but I am approving this PR.

File: app/components/views/GetStartedPage/hooks.js (hunk 2 above)
Comment (amassarwi) [reply app/components/views/GetStartedPage/hooks.js:218]: You are right and this was completely deleted in #3033 as it will run on each startup now!

**Pull request merged**